src/main.rs

```diff
@@ -1,8 +1,11 @@
+use chrono::{DateTime, Duration, FixedOffset, ParseError};
 use getopts::Options;
-use std::{collections::HashMap, error::Error, io::Write, num::ParseIntError, ops::AddAssign};
+use std::{
+    collections::HashMap, env, error::Error, fs, io::Write, num::ParseIntError, ops::AddAssign,
```

Comment:
How about changing the `env` import directly to `std::env::args()`? :wrench:

Reply:
Hmm, since there is only a single place calling that function, I don't think it is worth it.

Generally I'm not sure how I feel importing functions directly from other modules/libraries. I call them always with their partial namespace at least. 😅

Reply:
Okay, `env` should be rather removed from the import enumeration.  :wrench: